CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,2 @@
-OrionHDFSSink parameters are now called as "hdfs_*" instead of "cosmos_*" (#374)
+[FEATURE] OrionHDFSSink parameters are now called as "hdfs_*" instead of "cosmos_*" (#374)
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/c99ceb6385615cdb40b6d3db92a0d8fbb1214d0f Thanks!